### PR TITLE
Add greetings from amzing-gg

### PR DESCRIPTION
**Pull request by amzing-gg**
Hi!

### CODE REVIEW

File: pr-test.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 hello
 hi!
-// more texts
\ No newline at end of file
+Hello from amzing-gg
```
Comment (happyduck-git) [inline pr-test.html:3]:
```suggestion
Hello from "amzing-gg"
```

**Review by happyduck-git — CHANGES_REQUESTED** [review]
Please find my comments

**Review by amzing-gg — COMMENTED** [review]
I finished my review.

File: pr-test.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 hello
 hi!
-// more texts
\ No newline at end of file
+Hello from "amzing-gg"
```
Comment (happyduck-git) [inline pr-test.html:3]:
```suggestion
Hello from "amzing-gg"
Yay
```
I added more texts.

**Review by happyduck-git — CHANGES_REQUESTED** [review]

**Review by amzing-gg — COMMENTED** [review]
I reviewed again!

**Pull request merged**